raw-mm.dict.lua

```diff
@@ -3482,105 +3473,105 @@ dict = {
       end
     }
   },
-  lightcoltsfoot = {
-    physical = {
-      aspriority = 6,
-      spriority = 6,
-      balanceless_act = true,
-      herb = "coltsfoot",
-
-      isadvisable = function ()
-        -- disabled, handled with lightpipes by the system - but left here in case of manual lighting
-        return false
-      end,
-
-      oncompleted = function ()
-        pipes.coltsfoot.lit = true
-        sk.forcelight_coltsfoot = false
-        lastlit("coltsfoot")
-      end,
-
-      all = function ()
-        for name, pipe in pairs(pipes) do
-          pipe.lit = true
-          sk["forcelight_"..name] = false
-        end
-      end,
-
-      onstart = function ()
-        if conf.gagrelight then
-          send("light " .. pipes.coltsfoot.id, false)
-        else
-          send("light " .. pipes.coltsfoot.id, conf.commandecho) end
-      end
-    }
-  },
-  lightmyrtle = {
-    physical = {
-      aspriority = 5,
-      spriority = 5,
-      balanceless_act = true,
-      herb = "myrtle",
-
-      isadvisable = function ()
-        -- disabled, handled with lightpipes by the system - but left here in case of manual lighting
-        return false
-      end,
-
-      oncompleted = function ()
-        pipes.myrtle.lit = true
-        sk.forcelight_myrtle = false
-        lastlit("myrtle")
-      end,
-
-      all = function ()
-        for name, pipe in pairs(pipes) do
-          pipe.lit = true
-          sk["forcelight_"..name] = false
-        end
-      end,
-
-      onstart = function ()
-        if conf.gagrelight then
-          send("light " .. pipes.myrtle.id, false)
-        else
-          send("light " .. pipes.myrtle.id, conf.commandecho) end
-      end
-    }
-  },
-  lightfaeleaf = {
-    physical = {
-      aspriority = 4,
-      spriority = 4,
-      balanceless_act = true,
-      herb = "faeleaf",
-
-      isadvisable = function ()
-        -- disabled, handled with lightpipes by the system - but left here in case of manual lighting
-        return false
-      end,
-
-      oncompleted = function ()
-        pipes.faeleaf.lit = true
-        sk.forcelight_faeleaf = false
-        lastlit("faeleaf")
-      end,
-
-      all = function ()
-        for name, pipe in pairs(pipes) do
-          pipe.lit = true
-          sk["forcelight_"..name] = false
-        end
-      end,
-
-      onstart = function ()
-        if conf.gagrelight then
-          send("light " .. pipes.faeleaf.id, false)
-        else
-          send("light " .. pipes.faeleaf.id, conf.commandecho) end
-      end
-    }
-  },
+--   lightcoltsfoot = {
```

Comment:
Better to nuke it entirely to keep the code clean. We can always look back in git's history to pull it back.